components/UI/Map/GenericClusterGroup.tsx

```diff
@@ -20,7 +35,29 @@ const fetchIcon = (count: number) => {
 const emptyIcon =
   "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'/%3E";
 
-export const AhbapClusterGroup = ({ data }: Props) => {
+const DEFAULT_PROPERTY_MAP = {
+  name: "name",
+  description: "description",
+  type: "styleUrl",
+  icon: "icon",
+};
+
+const prepareProperties = (properties: any, propertyMap: any) => {
+  const preparedProperties = {};
```

Comment:
You can remove ts-ignore after you give a type to `preparedProperties`

`const preparedProperties: Record<string, any> = {};`